ci/ci.sh

```diff
@@ -317,7 +317,7 @@ build_sphinx_docs() {
       # TODO: revert to "make html" once "sphinx_panels" plugin is fully removed.
```

Comment:
nit: remove this?

Reply:
Removed the `make html` since we're already running `make develop`